cirq-core/cirq/ops/common_gates_test.py

```diff
@@ -720,6 +720,8 @@ def test_text_diagrams():
         cirq.CZ(a, b),
         cirq.CNOT(a, b),
         cirq.CNOT(b, a),
+        (cirq.CNOT ** 0.5)(a, b),
+        (cirq.CNOT ** 0.5)(b, a),
```

Comment:
Nit: can do apply then exponent, like for `H` below. IMO this is more readable:
```suggestion
        cirq.CNOT(a, b) ** 0.5,
        cirq.CNOT(a, b) ** 0.5,
```

Reply:
done